src/app/pages/account/view/[username].vue

```diff
@@ -1,18 +1,29 @@
 <template>
   <Loader :api="api" indicator="ping">
     <div class="flex flex-col gap-4">
-      <LayoutPageTitle :title="title" />
-      <div
-        class="flex min-w-0 flex-col items-center justify-center sm:flex-row"
-      >
-        <AccountProfilePicture
-          :account-id="account?.id"
-          class="size-48 rounded-sm"
-          height="192"
-          width="192"
-        />
-      </div>
-      <div class="flex justify-center">
+      <TypographyH2>
+        {{ title }}
+      </TypographyH2>
+      <div class="flex flex-col gap-4">
+        <div
+          class="light:border-(--semantic-base-line) flex flex-col items-center justify-center rounded-xl border border-1 sm:flex-row"
```

Comment:
I think that would be the first usage of `light:` across the codebase. Light is considered the default and we only add an override for `dark:`.

Also, how is `border` different from `border-1`?

```suggestion
          class="flex flex-col items-center justify-center rounded-xl border border-(--semantic-base-line) sm:flex-row dark:border-none"
```

Reply:
Ah it looked off in dark mode so I only added in light mode in the moment, should have added the opposite to dark.